src/controller/python/chip/clusters/attribute.cpp

```diff
@@ -359,17 +372,20 @@ chip::ChipError::StorageType pychip_ReadClient_ReadAttributes(void * appContext,
         ReadPrepareParams params(session.Value());
         params.mpAttributePathParamsList    = readPaths.get();
         params.mAttributePathParamsListSize = n;
-
+        params.mIsFabricFiltered            = pyParams.isFabricFiltered;
         if (pyParams.isSubscription)
         {
             params.mMinIntervalFloorSeconds   = pyParams.minInterval;
             params.mMaxIntervalCeilingSeconds = pyParams.maxInterval;
+            err                               = readClient->SendAutoResubscribeRequest(std::move(params));
+            SuccessOrExit(err);
+            readPaths.release();
```

Comment:
This is going to double-delete in the error case, no?  The release has to happen _before_ the SuccessOrExit.

In fact, I would do the release() right before the `SendAutoResubscribeRequest` both here and in ReadInteraction above.

Reply:
agree, just fixed it, thanks